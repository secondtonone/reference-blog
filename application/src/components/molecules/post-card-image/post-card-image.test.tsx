import renderer from 'react-test-renderer';
import PostCardImage from '.';

describe('Match snapshots PostCardContent', () => {
  it('default', () => {
    const tree = renderer.create(<PostCardImage
      url="url"
      previewImage="img"
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('horizontal', () => {
    const tree = renderer.create(<PostCardImage
      url="url"
      previewImage="img"
      horizontal
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('isLarge', () => {
    const tree = renderer.create(<PostCardImage
      url="url"
      previewImage="img"
      isLarge
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
