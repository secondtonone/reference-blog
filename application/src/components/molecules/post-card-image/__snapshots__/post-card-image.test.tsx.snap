// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots PostCardContent default 1`] = `
<div
  className="box-adaptive__BoxStyled-sc-g2wy39-0 VYInM post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
  data-test="image"
>
  <a
    data-test="image-link"
    href="url"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <img
      className="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
      data-src="img"
      height="190"
      src={null}
      width="158"
    />
  </a>
</div>
`;

exports[`Match snapshots PostCardContent horizontal 1`] = `
<div
  className="box-adaptive__BoxStyled-sc-g2wy39-0 lesazw post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
  data-test="image"
>
  <a
    data-test="image-link"
    href="url"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <img
      className="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
      data-src="img"
      height="190"
      src={null}
      width="158"
    />
  </a>
</div>
`;

exports[`Match snapshots PostCardContent isLarge 1`] = `
<div
  className="box-adaptive__BoxStyled-sc-g2wy39-0 eBllGr post-card-image__ImageStyled-sc-r39eru-0 fHpPev"
  data-test="image"
>
  <a
    data-test="image-link"
    href="url"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <img
      className="image__ImageStyled-sc-rj8okk-0 hueYwk lazyload"
      data-src="img"
      height="250"
      src={null}
      width="310"
    />
  </a>
</div>
`;
